cmd/gossamer/configcmd.go

```diff
@@ -254,7 +263,12 @@ func strToMods(strs []string) []api.Module {
 
 // dumpConfig is the dumpconfig command.
 func dumpConfig(ctx *cli.Context) error {
-	_, fig, err := makeNode(ctx, nil)
+	gen, err := loadGenesis(ctx)
```

Comment:
I don't think this should be necessary for dumpconfig, its just a dump of the cli commands

Reply:
okay I agree, I think this may need to be re written somewhat then, since it uses `makeNode` to get the config and `makeNode` will error if genesis is nil, what do you think a good move here is?

Reply:
If #342 gets merged, it refactors how we push the cli flags into the config. We could isolate those setup calls (eg. `setP2pConfig()`) to a containing function that is strictly responsible for setting up the config. This could then be called directly by `dumpConfig()` removing the need for genesis.

Reply:
okay I have changed it so that it no longer uses makeNode or genesis